fast/stages/01-resman/billing.tf

```diff
@@ -20,6 +20,8 @@ locals {
   # used here for convenience, in organization.tf members are explicit
   billing_ext_users = concat(
     [
+      module.branch-dp-dev-sa.iam_email,
```

Comment:
`teams` (see L32) has a dedicated local with both dev and prod SA emails, I'd use the same approach for consistency.